contracts/src/types/Price.sol

```diff
@@ -51,6 +51,8 @@ function ltB(AmountB x, AmountB y) pure returns (bool) {
 library PriceLib {
     using RayMathLib for *;
 
+    uint256 internal constant ONE_BPS = 1e6;
```

Comment:
this isn't 1BPS 1e6 we're were scaling by 1e6 would be 10k bips or 100%

Reply:
uniswap stores fees as 1 hundreth of a bip as the smallest unit so for example 100 fee units would be 1bps

Reply:
wrong name, will fix